### PR TITLE
Log stats

### PR DESCRIPTION
**Pull request by ghost**
implemented log stats in cpp

### CODE REVIEW

**Comment by ghost** [comment]
Können wir da morgen früh mal zusammen schauen ?

File: ros_ws/src/analysis-tools/src/log_stats.cpp [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+            << *m_speed_maxtime_it_max << delimiter << m_maxspeed_current << delimiter << m_maxspeed_delta << delimiter
+            << m_maxspeed_avg << delimiter << m_angle_current << delimiter << m_angle_delta << delimiter
+            << m_acceleration_current << delimiter << m_acceleration_smooth_mean << delimiter << m_acceleration_delta
+            << delimiter << m_acceleration_min << delimiter << m_acceleration_max << delimiter
+            << *m_acceleration_maxtime_it_max << delimiter << *m_acceleration_mintime_it_min << delimiter
+            << m_distance_current << delimiter << m_distance_delta << delimiter << m_turn_current << delimiter
+            << m_turn_delta << delimiter << m_speed_current * Config::RPM_FACTOR << std::endl;
+
+    filestream << logline.str();
+    filestream.close();
```
Comment (jhbruhn) [inline ros_ws/src/analysis-tools/src/log_stats.cpp:551]: Opening and closing the filestream every time we want to write something probably is not very efficient :) if we get performance issues again with this (cpu usage > 10% is what i'd call highly problematic), this has to be changed

Comment (ghost) [reply ros_ws/src/analysis-tools/src/log_stats.cpp:551]: I know ... I found a way today to open the files once via constructor. Also made some other stuff more efficient. Please recheck and merge

Comment (ghost) [reply ros_ws/src/analysis-tools/src/log_stats.cpp:551]: as an automatic benefit the file gets written buffered, so the fs wouldn't hyterize ;)

**Pull request merged**